jib-core/src/main/java/com/google/cloud/tools/jib/configuration/LayerConfiguration.java

```diff
@@ -70,29 +70,6 @@ public Builder addEntry(Path sourceFile, AbsoluteUnixPath pathInContainer) {
       return addEntry(sourceFile, pathInContainer, null);
     }
 
-    /**
-     * Adds an entry to the layer with the given permissions. Only adds the single source file to
-     * the exact path in the container file system. See {@link Builder#addEntry(Path,
-     * AbsoluteUnixPath)} for more information.
-     *
-     * @param sourceFile the source file to add to the layer
-     * @param pathInContainer the path in the container file system corresponding to the {@code
-     *     sourceFile}
-     * @param permissions the file permissions on the container. If null, then default permissions
-     *     are used (644 for files, 755 for directories)
-     * @param lastModified the modification time of the file
-     * @return this
-     * @see Builder#addEntry(Path, AbsoluteUnixPath)
-     */
-    public Builder addEntry(
```

Comment:
I don't think we should get rid of this.

Reply:
Yeah, I think I'll change things from the current `BuildConfiguration` approach to allow configuring different providers on `LayerConfiguration`. That said, I assume this should probably take a `FileTimestampProvider` instead of a `long`.